module/zfs/arc.c

```diff
@@ -3677,6 +3698,76 @@ arc_alloc_raw_buf(spa_t *spa, void *tag, uint64_t dsobj, boolean_t byteorder,
 	return (buf);
 }
 
+static void
+l2arc_hdr_arcstats_update(arc_buf_hdr_t *hdr, boolean_t incr,
+    boolean_t state_only)
```

Comment:
It looks like state_only is alway passed true. Do you have some future plans for this?